internal/services/applications/parse/permission_scope.go

```diff
@@ -76,3 +76,17 @@ func (id PermissionScopeId) Segments() []resourceids.Segment {
 func (id PermissionScopeId) String() string {
 	return fmt.Sprintf("Permission Scope (Application ID: %q, Scope ID: %q)", id.ApplicationId, id.ScopeID)
 }
+
+func (id PermissionScopeId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *PermissionScopeId) FromParseResult(input resourceids.ParseResult) error {
```